internal/users/users_test.go

```diff
@@ -279,6 +279,14 @@ func TestCleanUserFromLocalGroups(t *testing.T) {
 	}
 }
 
+func TestGetPasswdUsernames(t *testing.T) {
+	t.Parallel()
```

Comment:
I mentioned an unit test on purpose, as we are not going to export it, just have another file with this unit test in it, like `internal_test.go`.

Reply:
Damn, I really am rusty from the holidays hahahahaha

Reply:
see, you are even not making the difference between Rust and Go anymore! :p